support/fixtures/support.yaml

```diff
@@ -6160,3 +6160,52 @@
       queryable: false
       has_opacity: false
       default_opacity: null
+- model: support.layerfilter

```

Comment:
It have to be in an another fixure file.

Reply:
Created file support_layerfilter_fixtures.yaml.